components/blog-pagination.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+import {useRouter} from 'next/router'
+
+import colors from '@/styles/colors'
+import {ArrowLeftCircle, ArrowRightCircle} from 'react-feather'
+
+function listPages(num) {
+  const pageNumbers = []
+
+  for (let i = 0; i < num; i++) {
+    pageNumbers.push(i + 1)
+  }
+
+  return pageNumbers
+}
+
+function BlogPagination({pagination}) {
+  const router = useRouter()
+  const pageNumbers = listPages(pagination?.pages)
```

Comment:
Nous avions eu la discussion concernant la création directe d'un `Array` sans utiliser de `for loop` tout en respectant les recommandations de XO.

```suggestion
  const pageNumbers = Array.from({length: pagination.pages}, (v, i) => i + 1)
```